### PR TITLE
refactor(stepper): rework design and border

### PR DESCRIPTION
**Pull request by escapedcat**
Fixes:  #711

image

### CODE REVIEW

File: src/pages/Setup/Stepper.tsx [hunk 1]
```diff
@@ -10,25 +11,33 @@ export default function Stepper({ currentStep }: Props) {
   const { t } = useTranslation();
 
   return (
-    <div className="w-full">
-      <ul className="lg:flex hidden justify-center items-center gap-4">
+    <div
+      className={cn(
```
Comment (cstenglein) [inline src/pages/Setup/Stepper.tsx:15]: the `cn` function is not necessary here right?

File: src/pages/Setup/Stepper.tsx [hunk 2]
```diff
@@ -10,25 +11,33 @@ export default function Stepper({ currentStep }: Props) {
...
+        "w-full",
+        "after:block after:relative after:bottom-5 after:border after:border-gray-700 after:content-['']",
+      )}
+    >
+      <ul
+        className={cn(
```
Comment (cstenglein) [inline src/pages/Setup/Stepper.tsx:21]: same here

File: src/pages/Setup/Stepper.tsx [hunk 3]
```diff
@@ -10,25 +11,33 @@ export default function Stepper({ currentStep }: Props) {
...
+          "lg:flex hidden justify-between items-center relative z-10",
+        )}
+      >
         {Array.from({ length: NUMBER_OF_STEPS }).map((_, index: number) => (
-          <>
-            <li
-              className={`list-none text-center flex flex-col ${index < currentStep ? "opacity-100" : "opacity-30"}`}
-              key={index}
+          <li className={cn("list-none px-4 bg-primary-900")} key={index}>
```
Comment (cstenglein) [inline src/pages/Setup/Stepper.tsx:26]: same here

**Review by cstenglein — APPROVED** [review]

File: src/pages/Setup/Stepper.tsx (hunk 1 above)
Comment (escapedcat) [reply src/pages/Setup/Stepper.tsx:15]: I like to keep element-styles and `after`-styles separated

**Pull request merged**